lib/web/apiserver.go

```diff
@@ -396,7 +399,13 @@ func NewHandler(cfg Config, opts ...HandlerOption) (*APIHandler, error) {
 			session.XCSRF = csrfToken
 
 			httplib.SetNoCacheHeaders(w.Header())
-			httplib.SetIndexContentSecurityPolicy(w.Header())
+
+			// app access needs to make a CORS fetch request, so we only set the default CSP on that page
+			if strings.HasPrefix(r.URL.Path, "/web/launch") {
+				httplib.SetDefaultContentSecurityPolicy(w.Header())
```

Comment:
While retesting on https://doyensec2.teleport.sh/, I encountered this error
<img width="1615" alt="Screen Shot 2023-01-11 at 15 36 19" src="https://user-images.githubusercontent.com/6027823/211833915-21a07fa7-35a8-4019-8ec9-34d3fd5f0c65.png">

I believe this was fixed by this code, and my cluster is just old.